BootloaderCommonPkg/Include/Library/TpmLib.h

```diff
@@ -16,6 +16,17 @@
 
 #define ACPI_SSDT_TPM2_DEVICE_OEM_TABLE_ID  SIGNATURE_64('T', 'p', 'm', '2', 'T', 'a', 'b', 'l')
 
+/**
+  Extend SBL version in PCR [0].
+
+  @retval RETURN_SUCCESS      Operation completed successfully.
+  @retval Others              Unable to extend PCR.
+**/
+RETURN_STATUS
+TpmLogCrtmVersionEvent(
```

Comment:
It looks all the library API functions in this file, EFIAPI is missing. Please add it for both header file and implementation.